clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -3031,6 +3031,9 @@ def CIR_CallOp : CIR_CallOpBase<"call", [NoRegionArguments]> {
 
   let results = (outs Optional<CIR_AnyType>:$result);
   let arguments = commonArgs;
+  let regions = (region AnyRegion:$cleanup);
```

Comment:
I don't like this direction. I thought the incubator established that this was part of an insufficient design. Rather than attaching cleanups to every call that may throw an exception (resulting in duplicated cleanups that need to be merged later), I think we need to explicitly represent the cleanup scopes.

Reply:
I removed it and added a comment with a note until the redesign